guides/doc-Provisioning_Guide/topics/Bare_Metal.adoc

```diff
@@ -256,6 +256,140 @@ ifeval::["{build}" == "satellite"]
+endif::[]
+ifeval::["{build}" == "foreman"]
+. If you use the Katello plugin, click the *Parameters* tab, and ensure that a parameter exists that provides an activation key. If not, add an activation key.
+endif::[]
+. Click *Submit* to save the host details.
++
+For more information about network interfaces, see {BaseURL}/managing_hosts/index#Adding_Network_Interfaces[Adding network interfaces].
+
+. Set the host to boot in UEFI mode from network.
+
+. Start the host.
+
+. Select *Kickstart default PXEGrub2*.
+
+This creates the host entry and the relevant provisioning settings. This also includes creating the necessary directories and files for UEFI booting the bare metal host. When you start the physical host and set its boot mode to UEFI HTTP, the host detects the defined DHCP service and starts installing the operating system from its Kickstart tree.
+
+ifeval::["{build}" == "satellite"]
+When the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+ifeval::["{build}" == "foreman"]
+If you use the Katello plug-in, when the installation completes, the host also registers to {ProjectServer} using the activation key and installs the necessary configuration and management tools from the {project-client-name} repository.
+endif::[]
+
+.For CLI Users
+
+. On {Project}, update the `grub2-efi` package to the latest version:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {packages-update} grub2-efi
+----
+
+. Enable `foreman-proxy-http`, `foreman-proxy-httpboot`, and `foreman-proxy-tftp true` features.
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {installer-scenario} \
+--foreman-proxy-httpboot true \
+--foreman-proxy-http true \
+--foreman-proxy-tftp true 
+----
+
+. Create the host with the `hammer host create` command.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer host create --name "_My_Host_" \
+--organization "_My_Organization_" \
+--location "_My_Location_" \
+--hostgroup "_My_Host_Group_" \
+--mac "_aa:aa:aa:aa:aa:aa_" \
+--build true \
+--enabled true \
+--managed true \
+--pxe-loader "Grub2 UEFI HTTP"
+----
+
+. Ensure the network interface options are set using the `hammer host interface update` command.
++
+[options="nowrap" subs="+quotes"]
+----
+# hammer host interface update --host "_My_Host_" \
+--managed true \
+--primary true \
+--provision true
+----
+
+. Set the host to boot in UEFI mode from network.
+
+. Start the host.
+
+. Select *Kickstart default PXEGrub2*.
+
+This creates the host entry and the relevant provisioning settings. This also includes creating the necessary directories and files for UEFI booting the bare metal host. When you start the physical host and set its boot mode to UEFI, the host detects the defined DHCP service and starts installing the operating system from its Kickstart tree.
```

Comment:
The last sentence sounds like the Kickstart tree belonged to the dhcp service.
I'd probably put it in a way that DHCP service points the host to the HTTP endpoint of the satellite proxy with the KS tree (or something similar)

Reply:
Thanks for the review, I have implemented all your feedback in the last commit, please review.